### ISSUE TITLE
Secret length tooltip incorrectly warns for some secrets

### ISSUE THREAD

**Issue by aidanhs** [jsonwebtoken/jsonwebtoken.github.io#346]
For example, `9STK9w-taKnxxKuDqf2WuCQLz5EEbaQbIb_ah36YrCs`.

This appears to be because of the use of `.decode` https://github.com/jsonwebtoken/jsonwebtoken.github.io/blob/eae3210def118b2b0a073398719e8d2f3e52ac3f/src/editor/secret-length-tooltip.js#L20. Here's a nodejs repl session illustrating the issue:
```
$ node
> b64u = require('base64url')
{ [Function: encode]
  encode: [Circular],
  decode: [Function: decode],
  toBase64: [Function: toBase64],
  fromBase64: [Function: fromBase64],
  toBuffer: [Function: toBuffer],
  default: [Circular] }
> b64u.decode('9STK9w-taKnxxKuDqf2WuCQLz5EEbaQbIb_ah36YrCs').length
29
> b64u.encode(b64u.decode('9STK9w-taKnxxKuDqf2WuCQLz5EEbaQbIb_ah36YrCs')) == '9STK9w-taKnxxKuDqf2WuCQLz5EEbaQbIb_ah36YrCs'
false
> b64u.toBuffer('9STK9w-taKnxxKuDqf2WuCQLz5EEbaQbIb_ah36YrCs').length
32
> b64u.encode(b64u.toBuffer('9STK9w-taKnxxKuDqf2WuCQLz5EEbaQbIb_ah36YrCs')) == '9STK9w-taKnxxKuDqf2WuCQLz5EEbaQbIb_ah36YrCs'
true
```

As you can see, `.decode` has the wrong length and fails to round-trip the data. I think replacing with `.toBuffer` should fix it.

**Comment by aidanhs** [comment]
Ping, still relevant

**Comment by Sambego** [comment]
Fixed by #507, thanks for reporting this!
